### PR TITLE
Add a "getting started" guide for beginners

### PR DESCRIPTION
**Pull request by aviks**
The language of this page has been user-tested with high school student programmers. 

I haven't linked this page yet from other parts of our website, that can be a separate discussion. To begin with  (pun unintended) we need this to link from the Google Code In site. 

[Edit]. This text has been written by @xorJane  and myself.

### CODE REVIEW

**Comment by ViralBShah** [comment]
Looks good. Please merge whenever you think it is ready. We can link it prominently on the learning page.

File: learning/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+## Developing packages
+It is easy to move from using Julia packages to developing them. Most of our contributors started that way. If you are the user of a Julia package, please believe that you have the skills to contribute bug fixes and new features. Chris Rackaukas has created a video that walks you through the process of creating a new package, or contributing code to an existing package. Find it [here on our Youtube channel](https://www.youtube.com/watch?v=QVmU29rCjaA)
+
```
Comment (logankilpatrick) [inline learning/getting-started.md]:
```suggestion


## Looking for help

Programming is fun and exciting. The Julia programming language is no exception. While it is always best to try to answer a question yourself if you can, feel free to ask someone else more experienced! Take a look at the [Julia Community](https://julialang.org/community/) tab on the website to find all the places you can connect and get help in the Julia Community. 
```

Comment (logankilpatrick) [reply learning/getting-started.md]: May also be worth adding a link to the FAQ section of the docs: https://docs.julialang.org/en/v1/manual/faq/

Comment (logankilpatrick) [reply learning/getting-started.md]: Borrowing these ideas from: https://www.python.org/about/gettingstarted/

**Pull request merged**